src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs

```diff
@@ -63,7 +63,7 @@ internal static IEnumerable<WorkflowRoutesEntry> GetWorkflowRoutesEntries(Workfl
                 var activity = activityLibrary.InstantiateActivity<HttpRequestFilterEvent>(x);
                 var entry = new WorkflowRoutesEntry
                 {
-                    WorkflowId = workflow.WorkflowId,
+                    WorkflowId = workflow.Id.ToString(),
```

Comment:
Why this change?

Reply:
This is an old mistake, it couldn't work as somewhere else it is doing a `long.Parse()` on it. But `WorkflowId` is not a long, this is `Id` which is a long.

Here in `GetWorkflowRoutesEntries()` I did the same as in `GetWorkflowTypeRoutesEntries()`.

Reply:
Not sure I follow you. You know better. But maybe fix that is a separate PR

Reply:
Okay will remove asap

Reply:
Done